samples/musicfestival-frontend-react/src/components/Header.tsx

```diff
@@ -7,12 +7,14 @@ function Header() {
     const [isLoggedIn, setIsLoggedIn] = useState(false);
     const [username, setUsername] = useState("");
 
-    authService.getUser().then((user) => {
-        if (user && !user.expired) {
-            setIsLoggedIn(true);
-            setUsername(user.profile.name || "");
-        }
-    });
+    useEffect(() => {
```

Comment:
Everytime authService.getUser() is called, promise will be changed, which cause an infinite loop.
using useEffect() in this case will stop that behavior.